### ISSUE TITLE
Add support to `Install-ChocolateyFileAssociation` for multiple file extensions

### ISSUE THREAD

**Issue by philoushka** [chocolatey/choco#161]
Moving discussion from the old [chocolatey issue 683](https://github.com/chocolatey/chocolatey/issues/683) for function [Install-ChocolateyFileAssociation](https://github.com/chocolatey/chocolatey/blob/aa854645de9d0234d3017aad001e8f9bd8b46f2f/src/helpers/functions/Install-ChocolateyFileAssociation.ps1)

**Summary** 
- allow callers to send multiple file extensions instead of calling this method _n_ times for _n_ file extension associations.
- change param `string $extension` to `string[] $extensions`
- existing code will out in the wild will work because of parameter name disambiguation. `extension` is sufficient enough to be disambiguated to `extensions`, just as `ext` would too.
- change function to iterate through `$extensions` one by one and use existing code with no changes

**Comment by jberezanski** [comment]
A minor nitpick: PowerShell convention for parameter naming is to always use singular nouns, even in case of arrays. Therefore, I'd recommend against the name change.

**Comment by pauby** [comment]
`extension` should be able to take a string or an array of strings. I agree with @jberezanski, PowerShell uses singular nouns and the parameter shouldn't be renamed.

**Comment by pauby** [comment]
We should look to do this 8under #3477. 

@vexx32 changing from a string to a string array shouldn't cause any issues with current usage of it?

**Comment by vexx32** [comment]
Yeah, that should be relatively harmless 👍 while we're at it, we can have it accept a pipeline input maybe if it doesn't already have one, may as well.
